test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -1761,4 +1769,75 @@ public void testSealedDegradedClusterHealing() {
 
         assertThat(corfuRuntime.getLayoutView().getLayout()).isEqualTo(layout);
     }
+
+    /**
+     * Write a random entry to the specified CorfuTable.
+     *
+     * @param table CorfuTable to populate.
+     */
+    private void writeRandomEntryToTable(CorfuTable table) {
+        Random r = new Random();
+        corfuRuntime.getObjectsView().TXBegin();
+        table.put(r.nextInt(), r.nextInt());
+        corfuRuntime.getObjectsView().TXEnd();
+    }
+
+    /**
+     * Increment the cluster layout epoch by 1.
+     *
+     * @return New committed layout
+     * @throws OutrankedException If layout proposal is outranked.
+     */
+    private Layout incrementClusterEpoch() throws OutrankedException {
+        corfuRuntime.invalidateLayout();
+        Layout layout = new Layout(corfuRuntime.getLayoutView().getLayout());
+        layout.setEpoch(layout.getEpoch() + 1);
+        corfuRuntime.getLayoutView().getRuntimeLayout(layout).sealMinServerSet();
+        corfuRuntime.getLayoutView().updateLayout(layout, 1L);
+        return layout;
+    }
+
+    /**
+     * Test scenario where the sequencer bootstrap triggers cache cleanup causing maxConflictWildcard to be reset.
```

Comment:
Can you add a bit more description of steps to the test comments.

Reply:
Actually we don't seem to directly verify on reset maxConflicWildcard but instead making the conclusion from lack of exception and being able to do data operation.